python/rascaline/tests/utils/atomic_density.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose
+
+from rascaline.utils import GaussianDensity, LodeDensity
+
+
+pytest.importorskip("scipy")
+
+
+@pytest.mark.parametrize(
+    "density",
+    [
+        GaussianDensity(atomic_gaussian_width=1.2),
+        LodeDensity(atomic_gaussian_width=1.2, potential_exponent=1),
+    ],
+)
+def test_derivative(density):
+    # Exclude very small values due instabilitues of LodeDensity
```

Comment:
@kvhuguenin. This might be something we have to look into. I thought we avoid these instabilities by the expansion we do for small values but it seems not super stable...

Reply:
This was only a warning issued by `np.where`. I filter these warnings now.